Core/DISET/private/FileHelper.py

```diff
@@ -1,23 +1,18 @@
-# $HeadURL$
 __RCSID__ = "$Id$"
 
 import os
-try:
-  import hashlib
-  md5 = hashlib
-except:
-  import md5
-import types
+import hashlib as md5
```

Comment:
I would just do "import hashlib" and change md5->hashlib later in the code (4 cases) to definitely get rid of this historical indirection

Reply:
yes !! :)

Reply:
Done